nextage_ros_bridge/launch/nextage_ros_bridge_simulation.launch

```diff
@@ -1,16 +1,21 @@
 <launch>
+  <arg name="USE_IMPEDANCE" default="false" doc="true to enable Impedance Controller." />  <!-- Need to defined early to be used by another ags. -->
```

Comment:
https://github.com/tork-a/rtmros_nextage/pull/305#discussion_r118629938 is not entirely correct.

When I set this `USE_IMPEDANCE` to `true` in order to apply the `.wrl` file that has f/t sensors, "wait for HrpsysSeqStateROSBridge0 : None" error still occurs.

That said, what's causing the error above is the `main_ftsensor.wrl` file that I'm adding in this PR.